apps/web/views.py

```diff
@@ -55,3 +57,16 @@ def contact_us(request):
             response_data = {'message': 'Your message has been successfully recorded. We will contact you shortly.'}
             return Response(response_data, status=status.HTTP_201_CREATED)
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+@throttle_classes([AnonRateThrottle])
+@api_view(['GET'])
+@permission_classes((permissions.AllowAny,))
+def our_team(request):
+    team = Team.objects.all()
+    if not team:
```

Comment:
@yadavankit I don't think we need this check here. If there will be no team then an empty queryset will be returned and that is totally fine.